plugin/fe/src/api/generated/.gitignore

```diff
@@ -0,0 +1,3 @@
+wwwroot/*.js
```

Comment:
is content under `generated` folder required?

Reply:
deleted